code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/components/AepTextComposable.kt

```diff
@@ -108,12 +103,12 @@ internal fun AepTextComposablePreview() {
         AepTextComposable(AepText(content = "Multi\nLine\nText")) // Multi-line text
 
         // Null values for optional parameters
-        AepTextComposable(AepText(content = "Null Color", color = null))
-        AepTextComposable(AepText(content = "Null Alignment", align = null))
-        AepTextComposable(AepText(content = "Null Font", font = null))
+        AepTextComposable(AepText(content = "Null Color"))
+        AepTextComposable(AepText(content = "Null Alignment"))
+        AepTextComposable(AepText(content = "Null Font"))
 
         // Extreme font sizes
-        AepTextComposable(AepText(content = "Tiny Text", font = AepFont(size = 1)))
-        AepTextComposable(AepText(content = "Huge Text", font = AepFont(size = 100)))
+        AepTextComposable(AepText(content = "Tiny Text"))
+        AepTextComposable(AepText(content = "Huge Text"))
```

Comment:
Out of the scope of this PR: You can use `AepTextStyle` to create these previews in a following PR

Reply:
Sure